libs/backend/src/cast_vote_records/build_report_metadata.ts

```diff
@@ -219,15 +219,24 @@ export function buildCastVoteRecordReportMetadata({
         Name: `${election.state}`,
       },
     ],
-    vxBatch: batchInfo.map((batch) => ({
-      '@type': 'CVR.vxBatch',
-      '@id': batch.id,
-      BatchLabel: batch.label,
-      SequenceId: batch.batchNumber,
-      StartTime: batch.startedAt,
-      EndTime: batch.endedAt,
-      NumberSheets: batch.count,
-      CreatingDeviceId: generatingDeviceId,
-    })),
   };
 }
+
+/**
+ * Build that batch manifest that is included in the cast vote record export metadata file.
+ */
+export function buildBatchManifest({
+  batchInfo,
+}: {
+  batchInfo: Array<BatchInfo & { scannerId: string }>;
+}): CastVoteRecordBatchMetadata[] {
```

Comment:
I'm wondering whether we really need to transform the `BatchInfo` at all. We could just use `BatchInfo & { scannerId: string }` as the type instead of the `CastVoteRecordBatchMetadata` type, with its slightly differently named fields.

That said, there might be a reason for the field renaming that I'm missing!

Reply:
I think it's valuable to remap fields before serialization, otherwise you can end up serializing fields you don't want to in a spread operation. For example, `BatchInfo` has an `error` field which we don't want to include.

I also felt like the property names on `BatchInfo` were not sufficiently descriptive if I were to come across a `metadata.json` in the wild. E.g. `count` - count of what? Sheets or ballots? Or `endedAt` - ended at what? A number, or a time?

Reply:
Makes sense, thanks for clarifying!